components/DropdownLight/DropdownLight.unit.test.jsx

```diff
@@ -19,31 +19,32 @@ describe('<DropdownLight />', () => {
   it('should return a value from item, using a object items', () => {
     render(<DropdownLight items={itemsObjectMock} />);
 
-    const dropdown = screen.getByRole('combobox');
-    userEvent.selectOptions(dropdown, 'Lemon');
+    const dropdown = screen.getByRole('button');
```

Comment:
The test coverage shows that lines are not covered by tests

![image](https://user-images.githubusercontent.com/3389749/214876744-cb8ee128-fe78-4445-b7da-46c6f52e64e3.png)